src/debugpy/adapter/clients.py

```diff
@@ -215,15 +219,16 @@ def handle(self, request):
             if self.session.no_debug:
                 servers.dont_wait_for_first_connection()
 
+            request_options = request("debugOptions", json.array(str))
             self.session.debug_options = debug_options = set(
-                request("debugOptions", json.array(str))
+                
```

Comment:
Should `request_options` be passed in here?

Reply:
yeah this seems like a bug. The old code caused a type error I must have been in the middle of fixing it and didn't finish.